README.md

```diff
@@ -7,8 +7,8 @@
 
 ## ❗️Requirements
 
-- [ESLint](https://eslint.org/) `>= 6`
-- [Node.js](https://nodejs.org/) `>= 10`
+- [ESLint](https://eslint.org/) `>= 8`
```

Comment:
Dropping support for v7 entirely? How come?

Reply:
1. As a result of experiments, I found out that there is no ClassProperty selector in eslint 8 anymore.
2. Changed API for CliEngine.

But we can assign an additional selector that will work in v8, and the old one will work in v7. Also, since the use of CliEngine has been removed, we don't care what API is there.

Today, I will test in v7 version, and maybe I will return its support.